routing/osrm_helpers.hpp

```diff
@@ -0,0 +1,95 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include "osrm2feature_map.hpp"
+#include "osrm_data_facade.hpp"
+#include "osrm_engine.hpp"
+#include "routing_mapping.hpp"
+
+#include "indexer/index.hpp"
+
+namespace routing
+{
+namespace helpers
+{
+/// Class-getter for making routing tasks by geometry point.
```

Comment:
A helper class that prepares structures for OSRM by a geometry near a point.

?